LICENSE.txt

```diff
@@ -0,0 +1,132 @@
+
+    1. Human Rights Principles.
+
+        (a) Licensee is advised to consult the articles of the United Nations
+Universal Declaration of Human Rights and the United Nations Global Compact
+that define recognized principles of international human rights (the "Human
+Rights Principles"). Licensee shall use the Software in a manner consistent
+with Human Rights Principles.
+
+        (b) Unless the Licensor and Licensee agree otherwise, any dispute,
+controversy, or claim arising out of or relating to (i) Section 1(a) regarding
+Human Rights Principles, including the breach of Section 1(a), termination of
+this License for breach of the Human Rights Principles, or invalidity of
+Section 1(a) or (ii) a determination of whether any Law is consistent or in
+conflict with Human Rights Principles pursuant to Section 2, below, shall be
+settled by arbitration in accordance with the Hague Rules on Business and Human
+Rights Arbitration (the "Rules"); provided, however, that Licensee may elect
+not to participate in such arbitration, in which event this License (and all
+rights licensed hereunder) shall end immediately. The number of arbitrators
+shall be one unless the Rules require otherwise.
+
+        Unless both the Licensor and Licensee agree to the contrary: (1) All
+documents and information concerning the arbitration shall be public and may be
+disclosed by any party; (2) The repository referred to under Article 43 of the
+Rules shall make available to the public in a timely manner all documents
+concerning the arbitration which are communicated to it, including all
+submissions of the parties, all evidence admitted into the record of the
+proceedings, all transcripts or other recordings of hearings and all orders,
+decisions and awards of the arbitral tribunal, subject only to the arbitral
+tribunal's powers to take such measures as may be necessary to safeguard the
+integrity of the arbitral process pursuant to Articles 18, 33, 41 and 42 of the
+Rules; and (3) Article 26(6) of the Rules shall not apply.
+
+    2. Human Rights Laws. The Software shall not be used by any person or
+entity for any systems, activities, or other uses that violate any Human Rights
+Laws.  "Human Rights Laws" means any applicable laws, regulations, or rules
+(collectively, "Laws") that protect human, civil, labor, privacy, political,
+environmental, security, economic, due process, or similar rights; provided,
+however, that such Laws are consistent and not in conflict with Human Rights
+Principles (a dispute over the consistency or a conflict between Laws and Human
+Rights Principles shall be determined by arbitration as stated above).  Where
+the Human Rights Laws of more than one jurisdiction are applicable or in
+conflict with respect to the use of the Software, the Human Rights Laws that
+are most protective of the individuals or groups harmed shall apply.
+
+    3.	Indemnity. Licensee shall hold harmless and indemnify Licensor (and any
```

Comment:
Could fix spacing here while at it, compare to spacing off section 2 at line 74.

(I was just about to create a fix PR for this in the .md file)